rhizome/host/lib/vm_setup.rb

```diff
@@ -453,14 +462,12 @@ def cloudinit(unix_user, public_keys, gua, nics, swap_size_bytes, boot_image, dn
 domain-needed
 bogus-priv
 no-resolv
+cache-size=0
```

Comment:
this one is kind scary, caching is usually very good....do we absolutely need it? many operating systems even cache by default, CDNs have to deal with that.

Reply:
Removed this. Turns out, not necessary.